src/main/java/org/ambraproject/wombat/service/remote/UserApiImpl.java

```diff
@@ -53,16 +54,14 @@
     private final URL server;
     private final ImmutableCollection<BasicHeader> authorizationHeader;
 
-    private UserApiConfiguration(String server, String authorizationAppName, String authorizationPassword) {
+    private UserApiConfiguration(URI server) {
       try {
-        this.server = new URL(Objects.requireNonNull(server));
+        this.server = new URL(server.getScheme(), server.getHost(), server.getPort(), server.getPath());
       } catch (MalformedURLException e) {
         throw new RuntimeException(e);
       }
-
-      this.authorizationHeader = ((authorizationAppName != null) && (authorizationPassword != null))
-          ? ImmutableList.of(createAuthorizationHeader(authorizationAppName, authorizationPassword))
-          : ImmutableList.of();
+      String[] userInfo = server.getUserInfo().split(":");
```

Comment:
I wonder if it would be better to do this at bean init time (in `setNedUrl`), to surface missing user strings ASAP

Reply:
Yes, it probably would be.

Reply:
00183d387